### PR TITLE
Finally pluralization

### PR DESCRIPTION
**Pull request by dngroth**

### CODE REVIEW

**Review by sutser — DISMISSED** [review]

File: LinqToRest.Test/PluralizerTest.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿using Xunit;
+
+namespace Messerli.LinqToRest.Test
+{
+    public class PluralizerTest
```
Comment (tautropfli) [inline LinqToRest.Test/PluralizerTest.cs]: Seal this class

File: LinqToRest.Test/PluralizerTest.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        [Theory]
+        [MemberData(nameof(GetPluralizeData))]
+        public void PluralizesCorrect(string singular, string plural)
```
Comment (tautropfli) [inline LinqToRest.Test/PluralizerTest.cs]: Call this `expectedPlural`

**Review by tautropfli — CHANGES_REQUESTED** [review]

File: LinqToRest.Test/PluralizerTest.cs (hunk 2 above)
Comment (dngroth) [reply LinqToRest.Test/PluralizerTest.cs]: https://github.com/messerli-informatik-ag/linq-to-rest/pull/38/commits/3f1134df3175182b788de85153e1b4dbec0712ca

File: LinqToRest.Test/PluralizerTest.cs (hunk 1 above)
Comment (dngroth) [reply LinqToRest.Test/PluralizerTest.cs]: https://github.com/messerli-informatik-ag/linq-to-rest/pull/38/commits/3f1134df3175182b788de85153e1b4dbec0712ca

**Review by tautropfli — APPROVED** [review]

**Pull request merged**